internal/driver/fetch_test.go

```diff
@@ -126,23 +126,31 @@ func TestCollectMappingSources(t *testing.T) {
 
 func TestUnsourceMappings(t *testing.T) {
 	for _, tc := range []struct {
-		file, buildID, want string
+		os, file, buildID, want string
 	}{
-		{"/usr/bin/binary", "buildId", "/usr/bin/binary"},
-		{"http://example.com", "", ""},
+		{"", "/usr/bin/binary", "buildId", "/usr/bin/binary"},
+		{"", "http://example.com", "", ""},
+		{"windows", `C:\example.exe`, "", `C:\example.exe`},
+		{"windows", `c:/example.exe`, "", `c:/example.exe`},
 	} {
-		p := &profile.Profile{
-			Mapping: []*profile.Mapping{
-				{
-					File:    tc.file,
-					BuildID: tc.buildID,
+		t.Run("", func(t *testing.T) {
```

Comment:
Empty name for t.Run() is unusual. I'd pass tc.file or tc.file+"-"+tc.os (and use "any" instead of empty string to mean any OS).

Reply:
Ah, sorry, accidentally forgot to fill the name